gmso/tests/test_lammps.py

```diff
@@ -12,3 +14,33 @@ def test_write_lammps_triclinic(self, topology_site):
         top = topology_site()
         top.box = Box(lengths=[1,1,1], angles=[60,90,120])
         write_lammpsdata(top, filename='data.triclinic')
+
+    def test_water_lammps(self, water_system):
```

Comment:
I wish there was a nice way to compare these files when we write out, but with anything that deals with floating point math, its going to be a bit difficult to validate. 

I think we should consider/discuss other ways we can further validate our writers. But for now, I am happy with this.

Reply:
Yeah I agree.  Once the reader gets merged, I plan on using it to validate the actual contents of these files.